.github/workflows/cd.yml

```diff
@@ -271,16 +271,86 @@ jobs:
         path: dist
         name: dist
 
+  distribution-release:
+    name: Create easy-to-integrate distributions (release)
+    needs: [linux-x86_64, linux-i686, windows-x86_64, windows-i686, macos-x86_64, macos-arm64]
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: Download assets
+      uses: actions/download-artifact@v3
+      with:
+        name: dist
+    - name: Create commit-sha file
+      env:
+        GITHUB_SHA: ${{ github.sha }}
+      run: |
+        echo $GITHUB_SHA > dist/commit-sha
+    - name: Prepare distribution
+      run: |
+        cp dist/commit-sha distribution
+        cp ffi/webgpu-headers/webgpu.h distribution/include/webgpu
+        cp ffi/wgpu.h distribution/include/webgpu
+        for zipfile in wgpu-*-release.zip
+          arch=$(echo $zipfile | cut -d- -f2-3)
+          mkdir -p distribution/bin/$arch
+          unzip dist/wgpu-*-release.zip distribution/bin/
+          unzip -d distribution/bin $zipfile *.so *.dylib *.dll
+        done
+        mv distribution webgpu
+        zip -r wgpu-distribution.zip webgpu
+    - name: Upload
+      uses: actions/upload-artifact@v3
+      with:
+        path: wgpu-distribution.zip
+        name: dist-cmake
+
+  distribution-debug:
+    name: Create easy-to-integrate distributions (debug)
+    needs: [linux-x86_64, linux-i686, windows-x86_64, windows-i686, macos-x86_64, macos-arm64]
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: Download assets
+      uses: actions/download-artifact@v3
+      with:
+        name: dist
+    - name: Create commit-sha file
+      env:
+        GITHUB_SHA: ${{ github.sha }}
+      run: |
+        echo $GITHUB_SHA > dist/commit-sha
+    - name: Prepare distribution
+      run: |
+        cp dist/commit-sha distribution
+        cp ffi/webgpu-headers/webgpu.h distribution/include/webgpu
+        cp ffi/wgpu.h distribution/include/webgpu
+        for zipfile in wgpu-*-debug.zip
+          arch=$(echo $zipfile | cut -d- -f2-3)
+          mkdir -p distribution/bin/$arch
+          unzip dist/wgpu-*-debug.zip distribution/bin/
+          unzip -d distribution/bin $zipfile *.so *.dylib *.dll
+        done
+        mv distribution webgpu
+        zip -r wgpu-distribution.zip webgpu
+    - name: Upload
+      uses: actions/upload-artifact@v3
+      with:
+        path: wgpu-distribution.zip
+        name: dist-cmake
```

Comment:
Won't the zip-files for the debug and release distribution overwrite each-other?

Also, I think you can simply use `name: dist`, which keeps the GH release job simpler.

Reply:
Indeed, fixed!